### ISSUE TITLE
Are we removing relocatingDependents from visa job checks?

### ISSUE THREAD

**Issue by camerojo** [talentbeyondboundaries/tbbtalentv2#1420]
Now that we have copied this field to candidate opp, shouldn't we remove from visa job check?

**Comment by cazcam34** [comment]
Should already have been redundant in the code - double check to make sure it's safe to delete.

**Comment by cazcam34** [comment]
PR with flyway ready however there are 3 flyways ahead of mine, so wanted to wait for those to go through before pushing

**Comment by cazcam34** [comment]
Just waiting on flyway 326 to be merged into staging, before I create a PR with my 327 flyway.

**Comment by cazcam34** [comment]
My Flyway was merged in with the copyright flyway update #1736 - no PR needed for this
